src/internal/proc.js

```diff
@@ -626,6 +640,18 @@ export default function proc(
     channel.flush(cb)
   }
 
+  function runGetContextEffect(prop, cb) {
+    // TODO: if object returned -> return shallow copy?
+    // should yield getContext() be available to access context?
+    // if yes then shallow copying it is a must to avoid unexpected mutations
+    cb(taskContext[prop])
```

Comment:
Context property could be anything, not just plain object, so we can't safely copy it.